plugins/phrase-conector/src/plugin.tsx

```diff
@@ -30,6 +30,12 @@ registerPlugin(
         type: 'password',
         required: true,
       },
+      {
+        name: 'templateUId',
```

Comment:
Matching the Phrase API
```suggestion
        name: 'templateUid',
```